### ISSUE TITLE
Bootstrapping problem: lint call depends on ppx_driver

### ISSUE THREAD

**Issue by tautschnig** [janestreet/stdio#1]
The lint call (https://github.com/janestreet/stdio/blob/29ea76964b8dbcacb1397956040bc0b595bcfc71/src/jbuild#L8) makes jbuilder (1.0+beta18) look for ppx_driver.runner, but stdio itself is a transitive build dependency of ppx_driver.

I had to remove the lint lines to get stdio to build.

**Comment by ghost** [comment]
Are you using the development version of jbuilder? 1.0+beta18 hasn't been released in opam yet

**Comment by tautschnig** [comment]
Which version of jbuilder should I be using? I'm not using (I can't) opam.

**Comment by ghost** [comment]
You should use this one: https://github.com/ocaml/dune/releases/tag/1.0%2Bbeta17.

What error are you getting BTW? `(lint ...)` is only used by `jbuilder build @lint`, the normal build shouldn't be affected by this

**Comment by tautschnig** [comment]
I have [elsewhere](https://github.com/janestreet/ppx_expect/issues/7) been told that 1.0+beta17 is broken, hence I moved to beta18. But maybe the problem is just mine - I am getting

```
jbuilder build @install
Error: External library "ppx_driver.runner" not found.
-> required by src/jbuild
Hint: try: jbuilder external-lib-deps --missing @install
make: *** [default] Error 1
```
after I enabled additional debugging in jbuilder to figure out why it was reporting missing dependencies. Reverting to the original code it seems jbuilder remains silent about the missing dependency in the lint rule. Closing.
